public/src/components/channelManagement/epicTests/epicTestEditor.tsx

```diff
@@ -274,12 +269,6 @@ class EpicTestEditor extends React.Component<EpicTestEditorProps, TestEditorStat
 
         <hr />
 
-        <EpicTypeComponent
```

Comment:
I think we still need this until the migration is complete?
Can we leave it in for now, and remove at the same time as we add the channel to the menu?

Reply:
Ahhh yeah of course!